modules/nodes-cluster-resource-levels-command.adoc

```diff
@@ -44,44 +44,51 @@ spec:
 
 .Procedure
 
-To use the tool on the command line: 
+To use the tool on the command line:
 
-. Run the following command:
+. From the terminal, run:
++
+----
+podman login registry.redhat.io
```

Comment:
Missing a prompt "$" in front of the command. Also, terminal commands should specify `[source,terminal]` as the source language

Same for the below command too if you can update there as well

Reply:
Done